tests/test_modeling_tf_longformer.py

```diff
@@ -196,6 +199,41 @@ def create_and_check_longformer_for_question_answering(
         self.parent.assertListEqual(shape_list(result.start_logits), [self.batch_size, self.seq_length])
         self.parent.assertListEqual(shape_list(result.end_logits), [self.batch_size, self.seq_length])
 
+    def create_and_check_longformer_for_sequence_classification(
```

Comment:
```suggestion
    def create_and_check_for_sequence_classification(
```
Nit: we had decided to name those checks without the model name inside.